src/main/resources/tags/links/custom.tag

```diff
@@ -5,11 +5,14 @@ aliases: customblocks, customitems, skulls
 
 Geyser has support for custom items and custom blocks! Player heads can also be registered to show up in the Bedrock player inventory.
 
-For information on custom items, see https://wiki.geysermc.org/geyser/custom-items/.
-For custom blocks, see https://wiki.geysermc.org/geyser/custom-blocks/.
-For player heads, see https://wiki.geysermc.org/geyser/custom-skulls/.
+For information on custom items, see [here](https://wiki.geysermc.org/geyser/custom-items/).
+For custom blocks, see [here](https://wiki.geysermc.org/geyser/custom-blocks/).
+For player heads, see [here](https://wiki.geysermc.org/geyser/custom-skulls/).
 
-Converting glyphs or rank icons is also possible, but done manually. https://wiki.bedrock.dev/concepts/emojis.html should help to get started.
-Custom sounds are also possible - see https://wiki.bedrock.dev/concepts/sounds.html. Do note: Custom sounds should be registered under a custom namespace!
+Some conversions must be done manually.
+For converting glyphs or rank icons, see ["Custom Emojis"](https://wiki.bedrock.dev/concepts/emojis.html).
+For custom sounds, see ["Custom Sounds"](https://wiki.bedrock.dev/concepts/sounds.html). Do note: Custom sounds should be registered under a custom namespace!
+For converting armor textures, see ["Custom Armor"](https://wiki.bedrock.dev/items/custom-armor.html#top).
+JSON UI can be used to convert chest interfaces, scoreboard titles, and overall beautify your Minecraft Bedrock UI experience. See ["JSON UI"](https://wiki.bedrock.dev/json-ui/json-ui-intro.html) for more information.
```

Comment:
Personally, I'd change this into some sort of list/continuous text
e.g.: (feel free to adapt/change to your liking:)

```suggestion
Some conversions must be done manually. For example:
- [Converting glyphs or rank icons](https://wiki.bedrock.dev/concepts/emojis.html)
- [Custom sounds](https://wiki.bedrock.dev/concepts/sounds.html). Do note: Custom sounds should be registered under a custom namespace!
- [Armor textures](https://wiki.bedrock.dev/items/custom-armor.html).
- JSON UI can be used to create inventory menus, change scoreboard titles, and overall beautify your Minecraft Bedrock UI experience. See [JSON UI](https://wiki.bedrock.dev/json-ui/json-ui-intro.html) for more information.
```
Not quite sure how "scoreboard titles" would be noteworthy, maybe worth replacing this with something that is a bit more straightforward

Besides these, thanks for the contribution to the tags - always nice to have them up to date :p

Reply:
Im cool with these changes.